tests/integration_tests/full_test.sh

```diff
@@ -64,6 +64,7 @@ for call in \
   "$geoips_repopath/tests/scripts/ahi.tc.WV.geotiff.sh" \
   "$geoips_repopath/tests/scripts/ami.static.Infrared.imagery_annotated.sh" \
   "$geoips_repopath/tests/scripts/ami.static.Visible.imagery_annotated.sh" \
+  "$geoips_repopath/tests/scripts/ami.static.mst.absdiff-IR-BD.imagery_annotated.sh" \
```

Comment:
Again, does this have anything to do with IR-BD?

Reply:
No it doesn't. That's just the product I referenced when creating the new product. It uses the same variable so that's why I called it absdiff-IR-BD. In reality it can probably be absdiff-IR, or something along those lines.

Reply:
Yeah, doesn't matter! I'm just resolving this.